TelepresenceBot/mobile/src/main/res/drawable/ic_arrow_right.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Instead of copying these vectors 4 times, you can have 1 and rotate them in the view itself. Or we can also have a rotate drawable.